### ISSUE TITLE
Time Plugins: Cannot be loaded in RelWithDebInfo Mode

### ISSUE THREAD

**Issue by KerstinKeller** [eclipse-ecal/ecal#1251]
### Problem Description

When I am building eCAL in RelWitDebInfo mode, the created time plugins (which are suffixed `reldbg` are not being considered when loading the time plugins.
In Debug mode (suffic `d`) or Release mode (no suffix), the time plugins are loaded properly.

![image](https://github.com/eclipse-ecal/ecal/assets/13848742/40c05802-5641-4c19-bc19-ec990ca04d11)

### How to reproduce

Build eCAL in RelWithDebInfo Configuration and start `ecal_sample_person_snd`.
It prints
``` 
Could not load eCAL time sync module ecaltime-localtime.dll 
```

### How did you get eCAL?

Custom Build / Built from source

### Environment

eCAL Master

**Comment by KerstinKeller** [comment]
Indeed it behaves as described. We have two options:
If we're in RelWithDebInfo build, we search only / first dlls/so with the `reldbg` extension. 
So should we:
- only search for plugins with the "correct" extension?
- Search for mulitple plugins, and if so, in which order

**Comment by DownerCase** [comment]
What about not using custom postfixes? I.e: Delete this block of code https://github.com/eclipse-ecal/ecal/blob/f51d0c0d0d462e9be3ee6c8589b77253d02be892/CMakeLists.txt#L298-L302

(I guess you _could_ leave the debug postfix, as its _a_ thing some projects do...)

Otherwise you also have this problem with MinSizeRel too.

Release, MinSizeRel and RelWithDebInfo are all release builds after all, so makes some sense to not distinguish between them.

**Comment by KerstinKeller** [comment]
Yeah, not 100% sure why we have them in the first place. For `debug` we do, because we do often package debug and release artefacts in the same directories....

**Comment by FlorianReimold** [comment]
Exactly that's why we have those postfixes. They especially matter for Windows, where we have to provide special builds in Debug and Release mode. And if those shall go in the same directory, they must have different names, obviously. We maybe(?) can ditch the minsize and reldbg versions though, as I think the binary interface only differs between Debug and Non-Debug.

**Comment by DownerCase** [comment]
Resolved by #1990
